src/app/child-dev-project/attendance/add-day-attendance/roll-call/roll-call.component.html

```diff
@@ -1,6 +1,14 @@
 <!-- Individual Student's Page -->
 <div *ngIf="!isFinished() && entries?.length > 0">
-  <app-child-block [entity]="entries[currentIndex].child"></app-child-block>
+  <div class="attendance-progress-bar">
+    <mat-progress-bar
+      class="filler"
+      mode="determinate"
+      [value]="(currentIndex / entries.length) * 100"
+    ></mat-progress-bar>
+    <app-child-block [entity]="entries[currentIndex].child"></app-child-block>
+    <div class="align-right">{{ currentIndex }} / {{ entries.length }}</div>
```

Comment:
Could you move this to the same line as the `app-child-block` and align it to the right of the screen? 
Or maybe just have it in the same line but before the `app-child-block`, e.g. `3/12 Achyut Verma`.